### ISSUE TITLE
Make component pagination consistent with customer-admin-fe

### ISSUE THREAD

**Issue by colganmeanor** [justifi-tech/web-component-library#198]
The web component library had an API file consistent with the customer-admin-fe app until the introduction of pagination into the component library. We should update the component library pagination component and API file to be consistent with how the dashboard does it.

- [pagination component example](https://github.com/justifi-tech/customer-admin-fe/blob/main/src/shared/components/tables/PaginationComponent.tsx#L29)
- [applying params example](https://github.com/justifi-tech/customer-admin-fe/blob/main/src/shared/api/Api.ts#L52)

In short, I don't believe the API file should have to worry about managing and formatting the pagination params - it should just apply all params that are sent along to it.

**Comment by colganmeanor** [comment]
Work is completed to fix the pagination in the web component library, but the work is currently blocked by the following bug: 
https://justifi-ai.atlassian.net/browse/PAY-908?atlOrigin=eyJpIjoiMzNhNWJlMzExNDA5NDI1YWFmMWE5YmExYTFkMmM5ZDciLCJwIjoiaiJ9

The wc-proxy API is not accepting parameters correctly in API requests, thus pagination is not working in the components until this is fixed.

**Comment by colganmeanor** [comment]
Update: BE issued a quick fix for the params being passed through to the wc-proxy API. 

This work is now complete and ready for code review
